src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskType;
+import duke.task.ToDo;
+
+import static duke.task.TaskType.DEADLINE;
+import static duke.task.TaskType.EVENT;
+import static duke.task.TaskType.TODO;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        DukeList dList = new DukeList();
+        String[] greet = {getLogo(), "Hello! I'm Duke", "What can I do for you?"};
+
+        printOut(greet);
+        while (true) {
+            String cmd = scan.nextLine();
+            String[] arguments = cmd.split(" ");
+            try {
+                if (arguments[0].equals("bye")) {
+                    break;
+                } else if (arguments[0].equals("list")) {
+                    printOut(dList.listTasks());
+                } else if (arguments[0].equals("done")) {
+                    printOut(dList.markAsDone(Integer.valueOf(arguments[1])));
```

Comment:
To improve readability, perhaps can consider avoiding complicated expressions? In other words to split up this printOut instead of doing everything in 1 line.